v2/internal/app/app_production_obfuscated.go

```diff
@@ -0,0 +1,120 @@
+//go:build production && obfuscated
+
+package app
+
+import (
+	"context"
+
+	"github.com/wailsapp/wails/v2/internal/binding"
+	"github.com/wailsapp/wails/v2/internal/frontend"
+	"github.com/wailsapp/wails/v2/internal/frontend/desktop"
+	"github.com/wailsapp/wails/v2/internal/frontend/dispatcher"
+	"github.com/wailsapp/wails/v2/internal/frontend/runtime"
+	"github.com/wailsapp/wails/v2/internal/logger"
+	"github.com/wailsapp/wails/v2/internal/menumanager"
+	"github.com/wailsapp/wails/v2/pkg/options"
+)
+
+// App defines a Wails application structure
+type App struct {
+	frontend frontend.Frontend
+	logger   *logger.Logger
+	options  *options.App
+
+	menuManager *menumanager.Manager
+
+	// Indicates if the app is in debug mode
+	debug bool
+
+	// OnStartup/OnShutdown
+	startupCallback  func(ctx context.Context)
+	shutdownCallback func(ctx context.Context)
+	ctx              context.Context
+}
+
+func (a *App) Shutdown() {
+	a.frontend.Quit()
+}
+
+func (a *App) Run() error {
+	err := a.frontend.Run(a.ctx)
+	if a.shutdownCallback != nil {
+		a.shutdownCallback(a.ctx)
+	}
+	return err
+}
+
+// CreateApp creates the app!
+func CreateApp(appoptions *options.App) (*App, error) {
+	var err error
+
+	ctx := context.Background()
+
+	// Merge default options
+	options.MergeDefaults(appoptions)
+
+	debug := IsDebug()
+	ctx = context.WithValue(ctx, "debug", debug)
+
+	// Set up logger
+	myLogger := logger.New(appoptions.Logger)
+	if IsDebug() {
+		myLogger.SetLogLevel(appoptions.LogLevel)
+	} else {
+		myLogger.SetLogLevel(appoptions.LogLevelProduction)
+	}
+	ctx = context.WithValue(ctx, "logger", myLogger)
+	ctx = context.WithValue(ctx, "obfuscated", true)
```

Comment:
This file seems to be mostly redundant with `app_production.go`. Should we maybe just add a `IsObfuscated()` method whose value depends on the `obfuscated` build tag? We do the same with `IsDebug()`.